1756709725669	Sync.LogManager	DEBUG	Flushing file log
1756709725678	Sync.LogManager	DEBUG	Log cleanup threshold time: 1755845725678
1756709725679	Sync.LogManager	DEBUG	Done deleting files.
1756709728658	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1756709729971	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1756709731820	Sync.Service	DEBUG	User-Agent: Firefox/142.0 (Linux x86_64) FxSync/1.144.0.20250818081258.desktop
1756709731820	Sync.Service	INFO	Starting sync at 2025-09-01 08:55:31 in browser session YwKi1OBOWzel
1756709731820	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1756709731821	Sync.Status	INFO	Resetting Status.
1756709731821	Sync.Status	DEBUG	Status.service: error.sync.failed => success.status_ok
1756709731821	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1756709731821	Sync.SyncAuthManager	INFO	Getting sync key
1756709731821	FirefoxAccounts	DEBUG	getOAuthToken enter
1756709731821	FirefoxAccounts	TRACE	getCachedToken returning cached token
1756709731821	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1756709731821	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1756709731821	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1756709731821	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1756709731822	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1756709731822	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1756709731822	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1756709731823	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1756709731823	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1756709731823	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/open-uri
1756709731823	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1756709731823	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1756709731823	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/close-uri/v1
1756709731824	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/197106278/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1756709731824	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1756709731824	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1756709731824	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1756709731825	Sync.SyncScheduler	DEBUG	Sync error count has exceeded 3; enforcing backoff.
1756709731825	Sync.SyncScheduler	DEBUG	Starting client-initiated backoff. Next sync in 10059246 ms.
1756709731825	Sync.SyncScheduler	DEBUG	Next sync in 10059246 ms. (why=client-backoff-schedule)
1756709731825	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1756709731826	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
